prem_utils/connectors/openai.py

```diff
@@ -68,7 +68,7 @@ def chat_completion(
         self,
         model: str,
         messages: list[dict[str]],
-        max_tokens: int = None,
+        max_tokens: int = 128,
```

Comment:
OpenAI by default can get None, which is better

Reply:
Done